### DISCUSSION TITLE
How to increase Number of Max Listeners?

### DISCUSSION THREAD

**Discussion by BillyFKidney** [fuatakgun/eufy_security#698] [Q&A]
After adding door sensors throughout our home (Base Station v3 / S380) I'm receiving the a MaxListenersExceededWarning upon startup of the Eufy Add-On.

**How do I increase the number of listeners?**

I can see all the devices within the integrations, but the last four that were added are not updating:
<img width="913" alt="image" src="https://user-images.githubusercontent.com/65253277/213558233-81c0c115-0bd5-4e3f-8f72-c84e0d737c47.png">

`2023-01-19 20:54:03.651  INFO  Eufy Security server listening on host 0.0.0.0, port 3000 
(node:10) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 station talkback start listeners added to [EufySecurity]. Use emitter.setMaxListeners() to increase limit
(Use `node --trace-warnings ...` to show where the warning was created)
(node:10) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 station talkback stop listeners added to [EufySecurity]. Use emitter.setMaxListeners() to increase limit`

**Comment by RienduPre** [comment]
Same issue on my Homebase 2
